src/test/java/guitests/PanelRenameTest.java

```diff
@@ -130,4 +137,22 @@ public void panelRenameTest() {
         push(KeyCode.ENTER);
         sleep(EVENT_DELAY);
     }
+
+    @Test
+    public void panelRenameSelectOnlyOnePanel() {
+        // Test for the currently selected panel when trying to rename
+
+        pushKeys(MAXIMIZE_WINDOW);
+        while (panels.getPanelCount() < 3) {
+            pushKeys(CREATE_RIGHT_PANEL);
+        }
+        click("#dummy/dummy_col0_renameButton");
```

Comment:
Note that the two tests in this class will be done on the same window. This means that whichever test is executed later will have to work on panels created by the test that runs before it, and by JUnit standards [you have no guarantee as to which one will run first](http://stackoverflow.com/questions/3693626/how-to-run-test-methods-in-specific-order-in-junit4).

If `panelRenameTest` runs first, it will create 6 panels, which makes your test break as the first button you want to click is not even visible.

Off the top of my head, there are a couple of things you can do:
* Somehow slot the logic for this test into the currently-existing test: The test will be checking for *two* different things (the pre-existing test, as well as this PR's issue), which will not be obvious to future devs unless you make it clear which part of the test is testing what.
* Make use of Boards functionality to clear panels from the previous tests: You might have to deal with modal dialogs, which does not play well with Travis.
* Run `panelRenameSelectOnlyOnePanel` as a separate class: Not recommended as creating a new window consumes a lot of testing time, but if you find that other solutions are too difficult to implement then go ahead.